css/main.css

```diff
@@ -0,0 +1,258 @@
+*{
+    padding: 0;
+    margin: 0;
+    font-family: sans-serif;
+    font-weight: 100;
+    box-sizing: border-box;
+    
+}
+
+body{
+ background: #eceef1;
+}
+.sign-text{
+    font-size: 12px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    
+   
+}
+.sign-text p{
+    padding-bottom: 5px;
+}
+a{
+    color: #4faae8;
+    text-decoration: none;
+}
+.sign-in{
+    display: flex;
+    position:absolute;
+    top: 5px;
+    right: 10px;
+    justify-content: space-around;
+    color: #8d9096;
+    font-size: 9px;
+    align-items: center;
+}
+
+button img{
+    height: 20px;
+    background: white;
+    border: 0px;
+    border-radius: 100%;
+    
+    
+}
+
+button p{
```

Comment:
This style would not be needed. Element p should not be a child element of a button. Make the change in your HTML and remove these styles.

Reply:
Solved!